src/server/middleware/systraceMiddleware.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ * 
+ * @flow
+ */
+import type { Middleware, $Request } from 'express';
+
+const fs = require('fs');
+const dedent = require('dedent');
+const logger = require('../../logger');
+
+const systraceMiddleware: Middleware = (req: $Request, res, next) => {
+  if (req.path !== '/systrace') {
```

Comment:
I think it'll be better to do things like `app.use('/systrace', systraceMiddleware)` than having these checks in each middleware